packages/itwinui-react/src/core/Surface/Surface.tsx

```diff
@@ -36,44 +37,50 @@ const getSurfaceElevationValue = (elevation: SurfaceProps['elevation']) => {
 // ----------------------------------------------------------------------------
 // Surface.Header component
 
-export type SurfaceHeaderProps = React.ComponentPropsWithRef<'div'>;
+type SurfaceHeaderOwnProps = Record<never, never>;
```

Comment:
You could do `{}` with eslint disable line. Like in LinkAction.

Reply:
Changed